pkg/pillar/cmd/zedrouter/appcontainer.go

```diff
@@ -11,12 +11,17 @@ package zedrouter
 
 import (
 	"net"
+	"strconv"
 	"time"
 
+	"github.com/lf-edge/eve/pkg/pillar/iptables"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	log "github.com/sirupsen/logrus"
 )
 
+// DOCKERAPIPORT - constance define of docker API TCP port value
+const DOCKERAPIPORT int = 2375
+
 // check if we need to launch the goroutine to collect App container stats
 func appCheckStatsCollect(ctx *zedrouterContext, config types.AppNetworkConfig,
```

Comment:
Does this get called when the app instance is deleted? When it is deactivated?

Reply:
yes. when the config of AppInstance showing the GetStatsIPAddr is different from config to status, this is function is entered. the oldIPAddr if not nil, will trigger the remove of this ACL.

Reply:
That isn't the case I'm asking about.
Case 1: A delete of the AppInstanceConfig from zedagent. Does this function get called to remove the iptables rule?
Case 2: If Activate is set to false in the AppInstanceConfig should we remove the iptables rule? Or keep it around? (I don't know what we do to other iptables rules; making them behave the same seems the right answer.)

Reply:
Sorry, I did't realize you had added code to explicitly handle the delete case.